lib/irc/ClientPool.js

```diff
@@ -314,4 +315,15 @@ ClientPool.prototype._onJoinError = Promise.coroutine(function*(bridgedClient, c
     yield Promise.all(promises);
 });
 
+ClientPool.prototype._onNames = Promise.coroutine(function*(bridgedClient, chan, names) {
+    let mls = this._ircBridge.memberListSyncers[bridgedClient.server.domain];
+    if (!mls) {
+        log.info(`MemberListSyncer does not exist for ${bridgedClient.server.domain}: ` +
+            `not calling updateIrcMemberList and not syncing leaves (for channel ${chan}).`
```

Comment:
I wouldn't log this at all. This will be spammy (every time someone joins a channel a NAMES comes down) for users who aren't syncing membership lists.